lib/middlewares/responseGenerator.js

```diff
@@ -26,6 +26,10 @@
 var config = require('../../config'),
     synchronousRequests = {},
     events = require('events'),
+    logger = require('fiware-node-logger'),
```

Comment:
No. This project just hasn't migrated to logops yet. There was already an open issue about it #44

NTC